cwms-data-api/src/main/java/cwms/cda/data/dto/timeseriesprofile/ParameterInfo.java

```diff
@@ -0,0 +1,96 @@
+package cwms.cda.data.dto.timeseriesprofile;
+
+import java.util.Objects;
+
+import com.fasterxml.jackson.annotation.JsonInclude;
+import com.fasterxml.jackson.databind.PropertyNamingStrategies;
+import com.fasterxml.jackson.databind.annotation.JsonDeserialize;
+import com.fasterxml.jackson.databind.annotation.JsonNaming;
+import com.fasterxml.jackson.databind.annotation.JsonPOJOBuilder;
+import cwms.cda.api.errors.FieldException;
+import cwms.cda.data.dto.CwmsDTOBase;
+import cwms.cda.formatters.Formats;
+import cwms.cda.formatters.annotations.FormattableWith;
+import cwms.cda.formatters.json.JsonV2;
+
+@FormattableWith(contentType = Formats.JSONV2, formatter = JsonV2.class)
+@JsonDeserialize(builder = ParameterInfo.Builder.class)
+@JsonInclude(JsonInclude.Include.NON_NULL)
+@JsonNaming(PropertyNamingStrategies.KebabCaseStrategy.class)
+public class ParameterInfo implements CwmsDTOBase
+{
+	private final String parameter;
```

Comment:
The following columns are in the DB:
LOCATION_CODE, 
KEY_PARAMETER_CODE, 
PARAMETER_CODE, 
PARAMETER_UNIT, 
PARAMETER_FIELD, 
PARAMETER_COL_START, 
PARAMETER_COL_END

Reply:
Are you planning on updating to the full set of DB columns?

Reply:
yes. now that I know what col_start and col_end is used for

Reply:
added colStart and colEnd, note that those are exclusive to Field